src/core/dbus-cgroup.c

```diff
@@ -1882,15 +1882,15 @@ int bus_cgroup_set_property(
                 CGroupSocketBindItem **list;
                 uint16_t nr_ports, port_min;
                 size_t n = 0;
-                int family;
+                int family, ip_protocol;
```

Comment:
oh, actually the type for both these variables needs to be `int32_t`.  D-Bus only knows bit-width-specific types. 

Of course on all archs we support int and int32_t is the same type, but we should nonetheless use the right type for sd_bus_message_read(), as a matter of hygiene